apps/api/src/campaign-file/campaign-file.controller.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  Controller,
+  Get,
+  Post,
+  Body,
+  Response,
+  Param,
+  Delete,
+  StreamableFile,
+} from '@nestjs/common'
+import { CampaignFileService } from './campaign-file.service'
+import { Public } from 'nest-keycloak-connect'
+import { Logger, UseInterceptors, UploadedFiles } from '@nestjs/common'
+import { FilesInterceptor } from '@nestjs/platform-express'
+import { Multer } from 'multer'
+
+@Controller('campaign-file')
+export class CampaignFileController {
+  constructor(private readonly campaignFileService: CampaignFileService) {}
+
+  @Post(':campaign_id')
+  @Public()
```

Comment:
not sure what access we want to configure for this. I saw that creating a campaign is also public so I kept this the same.

Reply:
I think it should be allowed only for campaign coordinator, beneficiery or campaign admin roles